pyccel/codegen/printing/ccode.py

```diff
@@ -318,6 +318,12 @@ def _print_Module(self, expr):
                         imports = imports,
                         body    = body)
 
+    def _print_Break(self, expr):
+        return 'break;\n'
```

Comment:
Is the `\n` necessary? Isn't it added in `_print_CodeBlock`?

Reply:
yes it's not necessary